cli/test_gitconfig.ml

```diff
@@ -0,0 +1,23 @@
+let print_config fmt config =
```

Comment:
Shouldn't this file live in `test` instead of `cli`?

Reply:
Yeah, this was an ugly. Let me move it there.